pkg/api/vm/types.go

```diff
@@ -37,3 +37,7 @@ type AddVolumeInput struct {
 type RemoveVolumeInput struct {
 	DiskName string `json:"diskName"`
 }
+
+type ExportVolumeInput struct {
```

Comment:
This is unused. Set it to be the action input in the schema and parse it in the handler. Check `backupInput` as the example

Reply:
@gitlawr updated.